src/components/MarketList/index.js

```diff
@@ -121,14 +121,14 @@ class MarketList extends Component {
               </div>
             </div>
           </div>
-          <div className="info__group col-md-3">
+          {/* <div className="info__group col-md-3">
             <div className="info__field">
               <div className="info__field--icon icon icon--currency" />
               <div className="info__field--label">
                 <CurrencyName collateralToken={market.event.collateralToken} />
               </div>
             </div>
-          </div>
+          </div>*/}
```

Comment:
Can you remove commented out sections please? They will be in our git log if they're ever needed again